### PR TITLE
Profile resolution how-to readmes

### LINKED ISSUE #1178: Profile resolution how-to readmes
# Describe the bug

Examining improvements to profile resolution code (#1175) we are finding the readmes directing users in operating the profile resolver are defective.

The readmes explaining how to operate the XSLT profile resolver, with dependencies, limitations, and options, need to be updated.

# Who is the bug affecting?

Anyone who wants to use the profile resolver and can't figure it out for themselves.

# What is affected by this bug?

XSLT profile resolver is harder to (figure out how to) use.

# Expected behavior (i.e. solution)

Update the readmes to be more helpful.

### PR DESCRIPTION
**Pull request by nikitawootten-nist**
# Committer Notes

Fixes #1178

* Delete old `resolver-2018` folder

### All Submissions:

- [x] Have you selected the correct base branch per [Contributing](https://github.com/usnistgov/OSCAL/blob/main/CONTRIBUTING.md) guidance?
- [x] Have you set "[Allow edits and access to secrets by maintainers
](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)"?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/OSCAL/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [x] Do all automated CI/CD checks pass?

By submitting a pull request, you are agreeing to provide this contribution under the [CC0 1.0 Universal public domain](https://creativecommons.org/publicdomain/zero/1.0/) dedication.

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- ~Have you written new tests for your core changes, as applicable?~
- [x] Have you included examples of how to use your new feature(s)?
- ~Have you updated all [OSCAL website](https://pages.nist.gov/OSCAL) and readme documentation affected by the changes you made? Changes to the OSCAL website can be made in the docs/content directory of your branch.~

### CODE REVIEW

File: src/utils/util/resolver-pipeline/readme.md [hunk 1]
```diff
@@ -10,40 +10,46 @@ The sequence of XSLT transformations reflects and roughly corresponds to the ste
...
-Use a recent version of Saxon for best results &#8212; although we would also be *very interested* to hear from users of other XSLT engines conformant to the 3.1 family of XML standards (XSLT/XPath/XDM/XQuery).
+A serialized output of profile resolution takes the form of an OSCAL catalog. Assuming inputs are correctly formed, the output is valid to the catalog schema.
 
-The entry point for the transformation pipeline is `oscal-profile-RESOLVE.xsl`, which invokes the transformation steps in sequence, taking the source profile document as primary input. Load Saxon with your document and this stylesheet as follows (for example):
+## Invoking the Pipeline
 
-```bash
->  java -cp saxon-he-10.0.jar net.sf.saxon.Transform -t -s:YOUR_PROFILE_DOCUMENT.xml -xsl:path/to/oscal-profile-RESOLVE.xsl -o:YOUR_RESULT_BASELINE.xml
-```
+As of now, the profile resolution pipeline **is incompatible with Saxon versions newer then 10.x**. For best results, use Saxon 10, though you may have luck with other XSLT engines conformant to the 3.1 family of XML standards (XSLT/XPath/XDM/XQuery).
```
Comment (aj-stein-nist) [inline src/utils/util/resolver-pipeline/readme.md:34]: Blocking: if we are going to merge the resolver fix in #1639 (heads up, I definitely intend to ASAP), I would rather not leave documentation we will need to update right after.

```suggestion
The profile resolver and other utilities in this repo rely on open-source libraries and utilities developed by third parties. To review the software version used by this project, please review [the `build` directory for manifests of software and the specific versions](../../../../build) we recommend for running utilities from this repo, specifically the recommended [version of Saxon XSLT processor defined in the pom.xml manifest](../../../../build/pom.xml).
```

**Review by aj-stein-nist — CHANGES_REQUESTED** [review]
Excellent work, Nikita and Wendell. I would request one small change to be kind to our future selves.

File: src/utils/util/resolver-pipeline/readme.md (hunk 1 above)
Comment (nikitawootten-nist) [reply src/utils/util/resolver-pipeline/readme.md]: @aj-stein-nist If this is the case I will also add the wrapper script to this commit along with updating the readme. Please hold before merging.

**Comment by aj-stein-nist** [comment]
I updated the title because this one should be ready to merge soon with the #1639 replacement, #1685.

**Review by aj-stein-nist — APPROVED** [review]

**Pull request merged**